servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -2259,6 +2286,13 @@ void TextureStorage::render_target_set_as_unused(RID p_render_target) {
 	rt->was_used = false;
 }
 
+void TextureStorage::render_target_set_msaa(RID p_render_target, RS::ViewportMSAA p_msaa) {
+	RenderTarget *rt = render_target_owner.get_or_null(p_render_target);
+	ERR_FAIL_COND(!rt);
```

Comment:
can we make an early return here?
```
if (rt->msaa == p_msaa) {
	return;
}
```

Reply:
You're right, we could, but I wasn't done for all the other render_target_set_xxx methods so I'd rather do it in a refactoring PR later.

Reply:
This one needs the early return because it calls a function. The others don't call functions so they don't need early returns.